src/main/java/com/puppycrawl/tools/checkstyle/PackageNamesLoader.java

```diff
@@ -187,7 +187,7 @@ private static void processFile(URL packageFile, PackageNamesLoader namesLoader)
      *
      * @return map between local resources and dtd ids.
      */
-    private static Map<String, String> createIdToResourceNameMap() {
+    public static Map<String, String> createIdToResourceNameMap() {
```

Comment:
This case looks not possible to cover. But to cover the test it is required to make it public.

Please give your suggestion if it is not fine